gem/gem.py

```diff
@@ -782,6 +783,58 @@ def __new__(cls, i, j):
         return self
 
 
+class Inverse(Node):
+    """ Inverse is a shape to shape operation.
+
+        :arg aggregate: Tensor to be inversed
+        :arg multiindex: Indices for result of Inverse
+
+        Inverse is different from ComponentTensor, because its children must have shape.
+        It does no transformation between free indices and shape, but the multiindex is still necessary.
+        The multiindices of the children are used for indexing them when passing to inverse function call,
+        the multiindex of the Inverse node itself is used to index the result of the function call.
```

Comment:
Still these outdated multiindex comments...

Reply:
It is going to be more future-proof to just clearly explain what `Inverse` is, rather than how it related to `ComponentTensor`.  (What does `Inverse` have to do with `ComponentTensor` at all?)